com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs

```diff
@@ -313,31 +306,28 @@ public void TestAgent()
 
             agent1.LazyInitialize();
 
-            var numberAgent1Reset = 0;
+            var numberAgent1Reset = 1; // agent1 resets when initialized
```

Comment:
nit: now that the method isn't called reset(), can you rename some of these variables? And things like `agent2.agentResetCalls`? We probably understand it now, but someone new to the codebase wouldn't.